mitiq/calibration/calibrator.py

```diff
@@ -181,7 +181,9 @@ def get_cost(self) -> Dict[str, int]:
             A summary of the number of circuits to be run.
         """
         num_circuits = len(self.problems)
-        num_options = len(self.settings.technique_params)
+        num_options = 0
+        for strategy in self.settings.make_strategies():
+            num_options += strategy.num_circuit_required()
```

Comment:
Strategies get stored in each `Calibrator` instance, so no need to regenerate them here.
```suggestion
        num_options = sum(
            strategy.num_circuit_required() for strategy in self.strategies
        )
```